src/app/components/articles-grid.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { usePostQuery } from '../store/hooks/usePostQuery';
+import Article from './article';
+
+const ArticlesGrid = () => {
+    const postQuery = usePostQuery({ page: 0, per_page: 10 });
+
+    if ( postQuery.status === 'pending' ) {
+        return <em>Loading</em>;
```

Comment:
Always use brackets.
```suggestion
        return (
        	<em>Loading</em>
        );
```

Reply:
It's just one element ( it's gonna be. e.g. `<Loading />` ), not a big deal.